doc/introduction/data.rst

```diff
@@ -0,0 +1,112 @@
+.. role:: html(raw)
+   :format: html
+
+.. _intro_ref_data:
+
+Quantum Datasets
+================
+
+PennyLane provides the :mod:`~.data` subpackage to download, create, store and manipulate quantum datasets within the PennyLane framework.
+
+Loading Datasets in PennyLane
+-----------------------------
+
+We can access data of a desired type with the :func:`~pennylande.data.load` function. This downloads the desired
+dataset or loads it from local storage if previously downloaded. Some examples of data categories available for download include 'qchem',
+quantum chemistry-related datasets, and 'qspin', spin-system-related datasets. To specify the desired data, we use several arguments.
+For 'qchem' this includes the molname, basis, and bondlength, and for 'qspin' this includes sys_name, periodic, lattice, and layout. 
```

Comment:
```suggestion
dataset or loads it from local storage if previously downloaded.

To specify the dataset to be loaded, the data category (``data_type``) must be
specified, alongside category-specific keyword arguments. For the full list
of available datasets, please see the dataset library documentation.
```

Reply:
@DSGuala a suggestion here to make it more generic, and link back to the QML pages Utkarsh is putting together

Reply:
Thanks, suggestion committed.